packages/jcustomer/migrations/migrate-to-v11.yaml

```
@@ -39,3 +40,21 @@ actions:
     install:
       jps: "${baseUrl}/../update-events.yml"
 
+  configureShardsAndReplicas:
+    - setGlobals:
+        shards_number: 1
+        replicas_number: 1
+    - cmd [${nodes.cp.first.id}]: |
+        [[ "$envmode" = "production" ]] && echo true || echo false
+        exit 0
+    - if (${response.out}):
+      - setGlobals:
+          shards_number: 2
+    - env.control.AddContainerEnvVars [cp]:
+      vars:
+        UNOMI_ELASTICSEARCH_DEFAULTINDEX_SHARDS: ${globals.shards_number}
+        UNOMI_ELASTICSEARCH_MONTHLYINDEX_SHARDS: ${globals.shards_number}
+        UNOMI_ELASTICSEARCH_DEFAULTINDEX_REPLICAS: ${globals.replicas_number}
+        UNOMI_ELASTICSEARCH_MONTHLYINDEX_REPLICAS: ${globals.replicas_number}
+    - cmd [cp]: |-
+        sed -iE '/UNOMI_ELASTICSEARCH_DEFAULTINDEX_SHARDS|UNOMI_ELASTICSEARCH_MONTHLYINDEX_SHARDS|UNOMI_ELASTICSEARCH_DEFAULTINDEX_REPLICAS|UNOMI_ELASTICSEARCH_MONTHLYINDEX_REPLICAS/d' /opt/jcustomer/jcustomer/bin/setenv
```

I think a "service karaf restart" is required here. Otherwise, changes won't be take into account

---

Given the EC migration in the migration manifest, I think it is best to restart the karaf at this time.

Nevertheless, this bothers me:
```suggestion
        sed -iE '/UNOMI_ELASTICSEARCH_(DEFAULT|MONTHLY)INDEX_(SHARDS|REPLICAS)/d' /opt/jcustomer/jcustomer/bin/setenv
```